db/seeds.rb

```diff
@@ -84,6 +84,18 @@
   grouping.invite([student.user_name],
                   StudentMembership::STATUSES[:inviter],
                   invoked_by_admin=true)
+  group = Group.new
+  group.group_name = "#{student.user_name} a2"
+  group.save
+  grouping = Grouping.new
+  grouping.group = group
+  grouping.assignment = a2
+  grouping.save
+  (0..1).each do |count|
+    grouping.invite([students[time + count * 15].user_name],
+                     StudentMembership::STATUSES[:inviter],
+                     invoked_by_admin=true)
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Surrounding space missing for operator '='.